crypto/evp/pmeth_lib.c

```diff
@@ -852,7 +865,19 @@ static int legacy_ctrl_to_param(EVP_PKEY_CTX *ctx, int keytype, int optype,
             ERR_raise(ERR_LIB_EVP,
                       EVP_R_OPERATION_NOT_SUPPORTED_FOR_THIS_KEYTYPE);
             return -2;
+        case EVP_PKEY_CTRL_RSA_KEYGEN_BITS:
+            return EVP_PKEY_CTX_set_rsa_keygen_bits(ctx, p1);
+        case EVP_PKEY_CTRL_RSA_KEYGEN_PUBEXP:
+            return EVP_PKEY_CTX_set_rsa_keygen_pubexp(ctx, p2);
+        case EVP_PKEY_CTRL_RSA_KEYGEN_PRIMES:
+            return EVP_PKEY_CTX_set_rsa_keygen_primes(ctx, p1);
         }
+    case EVP_PKEY_CTRL_RSA_KEYGEN_BITS:
```

Comment:
I have no idea how this compiles...
Did you really mean to have this 2nd set of cases for RSA_KEYGEN_XXX ??

Reply:
This may resolve itself in a rebase I'm currently doing.

Reply:
Yup, gone.  It was a rebase resolution that didn't happen (i.e. no conflict), and I obviously forgot to check that it built.